lib/GeoIP2/WebService/Client.pm

```diff
@@ -35,11 +35,12 @@ use namespace::clean -except => 'meta';
 
 with 'GeoIP2::Role::HasLocales';
 
-has user_id => (
+has account_id => (
     is       => 'ro',
     isa      => MaxMindID,
     required => 1,
 );
+*user_id = \&account_id;    # for backwards-compatibility
```

Comment:
We used to support a public `user_id` function. This makes sure that still exists and that the new `account_id` method gets called when anyone calls the old `user_id` method that would otherwise have been removed by renaming the attribute.

Reply:
👍